src/FilamentManager.php

```diff
@@ -196,13 +196,13 @@ public function version()
         return InstalledVersions::getPrettyVersion('filament/filament');
     }
 
-    public static function ignoreMigrations()
+    public function ignoreMigrations()
     {
-        static::$shouldRunMigrations = false;
+        $this->shouldRunMigrations = false;
     }
 
-    public static function runsMigrations()
+    public function runsMigrations()
```

Comment:
```suggestion
    public function shouldRunMigrations()
```

Change the method name here - it's a naming convention used elsewhere in Filament.